docs/source/examples/plots_examples.rst

```diff
@@ -1,6 +1,182 @@
 Plotting with manim
 =================================
 
+Examples to illustrate the use of GraphScenes in manim
+
+
+.. manim:: Plot1
+    :quality: medium
+    :save_last_frame:
+
+    class Plot1(GraphScene):
+        def construct(self):
+            self.setup_axes()
+            func_graph=self.get_graph(lambda x: np.sin(x))
+            self.add(func_graph)
+
+.. manim:: Plot2yLabel
+    :quality: medium
+    :save_last_frame:
+
+    class Plot2yLabel(GraphScene):
+        CONFIG = {
+            "y_min": 0,
+            "y_max": 100,
+            "y_tick_frequency": 10,
```

Comment:
`y_tick_frequency` does not seem to exist, is it possible that this should be something like

```suggestion
            "y_axis_config": {"tick_frequency": 10},
```

?